docs/developer/pandas/base.rst

```diff
@@ -0,0 +1,12 @@
+BasePandasDataset
```

Comment:
I am not sure of this document should be put here. Why don't we want to put it into `docs/flow/modin/pandas/`?
cc @devin-petersohn

Reply:
The branch is based on #3000. It is better to discuss it there:)